features/neutron/rpms/config.pan

```diff
@@ -1,7 +1,8 @@
 unique template features/neutron/rpms/config;
 
-prefix '/software/packages';
-'{openstack-neutron}' ?= dict();
-'{python3-neutronclient}' ?= dict();
-'{ebtables}' ?= dict();
-'{ipset}' ?= dict();
+'/software/packages' = {
+    pkg_repl('openstack-neutron');
+    pkg_repl('python3-neutronclient');
+
+    SELF;
```

Comment:
Not needed, but harmless.

Reply:
Not sure to understand the comment... Is it about `SELF;`? I tend to use always use it even if in some cases, like this it is not needed...